src/main/webapp/WEB-INF/views/registration.jsp

```diff
@@ -52,16 +50,20 @@
         <label for="username">User name</label>
     </div>
     <div class="form-label-group">
-        <form:input path="login" id="login" class="form-control" placeholder="Login name" required="required"
-                    autofocus=""/>
-        <label for="login">Login name</label>
+         <form:input path="login" id="login" class="form-control is-invalid" placeholder="Login name" required="required"
```

Comment:
It shows validation error even if I just opened the page. It confuses me as a user :)